### PR TITLE
Consolidated Doc Update

### LINKED ISSUE #87: slightly different behavior on empty export
Minor change in handling the return output for exports.  See https://github.com/CruxConnect/crux-ui/issues/558  for details.

Will change at release of https://github.com/CruxConnect/crux-api/issues/858

### LINKED ISSUE #89: Create skus and items incomplete
There are two endpoints that currently show incomplete information: create items and create skus. Specifically the pricing data is missing. There could be more missing.

### LINKED ISSUE #105: Measurements verbiage is incorrect
The SKU Measurements Object and the Package Measurements Object have incorrect verbiage in them in describing units.

On input, the units may be be whatever the supplier wants (of the available options) with the default as 'g' / 'cm'.  On output, the units will always be metric ('g'/'cm') (at least for the API and until we add additional options for requesting different units).

Getting the wording right on this is probably going to be tricky.

Please consult @edje-jeter and @jtprince to ensure you've got it correct.

### LINKED ISSUE #115: Review and Update Product - Supplier
Continue review and Update of Product - Supplier Section.

Product -Supplier was partially completed ... placed on hold pending completion of Discovery Catalog.  Ref: #116 

We also need to review moving applicable Product docs to Supplier section.

### LINKED ISSUE #117: Password tokens expire in 2 hours
Add a note to the password token endpoints that a new token will expire in 2 hours.

### PR DESCRIPTION
**Pull request by nickcoleman**
Update a consolidation of issues

- [x] #87 Different behavior on empty export - returns a csv file with empty values
- [x] #89 Update Create Items and Create SKUs
- [x] #105 Measurements verbiage is incorrect
- [x] #117 Password tokens expire in 2 hours
- [x] #115 Review and Update Product - Supplier

fixes #87
fixes #89
fixes #105
fixes #115
fixes #117

### CODE REVIEW

File: _documentation/authentication.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ right_code: |
 You need to be authenticated for all API requests. In order to get authenticated, you must first retreive your API token.
 This can be done by <!--logging into to the UI and visiting account details, or --> sending a
 [retreive auth token](#organizationretreive_auth_token) request.
-The authentication token you receive is to be provided as a header in subsequent calls.
+The authentication token you receive is to be provided as a header in subsequent calls. (note: authentication tokens expire two hours after being issued)
```
Comment (Gidgidonihah) [inline _documentation/authentication.md]: This is untrue. If my task was unclear, I apologize. Auth tokens themselves don't every expire (so far as I'm aware... @jfischerlaunch ?). It's the reset and invitation tokens that expire. So on resetting your password, the token that will be received in the email. That will expire. And I believe that is on an internal endpoint.

File: _organization/retreive_auth_token.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ password
 ### Response Parameters:
 
 auth_token
-: (string) Authentication Token to be used for subsequent API calls
+: (string) Authentication Token to be used for subsequent API calls (note: authentication tokens expire two hours after being issued)
```
Comment (Gidgidonihah) [inline _organization/retreive_auth_token.md]: Some not as above about it not being the auth token that expires, but the reset/invite token.

File: _product_supplier/create_sku.md [hunk 3]
```diff
@@ -106,35 +111,20 @@ quantity_on_backorder
...
 
-number_of_units_bundled
-: (number) The Number of Units Bundled parameter indicates how many SKUs are in a single bundle.
+msrp
+: (number) The Manufacturer's Suggested Retail Price for the SKU. This is only a suggestion. It is not a price floor nor is it a price ceiling.
 
-<!--
-task github https://github.com/CruxConnect/crux-api/issues/1017  Backend doesn't support passing in objects or arrays of objects
 
+<!-- To Be Implmented
```
Comment (Gidgidonihah) [inline _product_supplier/create_sku.md]: What does this mean? Is there an open issue for it?

Let's be more specific or at least link an issue with specifics.

Comment (nickcoleman) [reply _product_supplier/create_sku.md]: John tells me the fields are in his development plan.  But, there is no specific crux-api issue to reference.  I've created one for the docs (#150) and reference it in the paw file.  I removed the TBD params from the create sku markup file.

**Pull request merged**